Sparkle/hr.lproj/Sparkle.strings

```diff
@@ -78,16 +78,16 @@
 "Ready to Install" = "Spremno za instaliranje";
 
 /* Message that is optionally shown at startup to allow users to turn on/off update checks. */
-"Should %1$@ automatically check for updates? You can always check for updates manually from the %1$@ menu." = "Da li da %1$@ automatski provjerava nadogradnje? Nadogradnje možeš i ručno provjeriti u %1$@ izborniku.";
+"Should %1$@ automatically check for updates? You can always check for updates manually from the %1$@ menu." = "Želiš li da %1$@ automatski provjerava nadogradnje? Nadogradnje možeš i ručno provjeriti u %1$@ izborniku.";
 
-"The update is improperly signed." = "Nadogradnja ima neispravnu signaturu.";
+"The update is improperly signed." = "Nadogradnja ima neispravan potpis.";
 
 "Update Error!" = "Greška prilikom aktualiziranja!";
 
-"Updating %@" = "Aktualiziranje %@";
+"Updating %@" = "Aktualiziranje %@a";
```

Comment:
That `a` doesn't seem right. If you're trying to give it a grammatical form, that may not work well with all app names, so it'd be better to find a different phrasing that doesn't require it.

Reply:
Thanx, you're right, there is a better way. I'll change it.